velox/exec/tests/TableWriteTest.cpp

```diff
@@ -1991,6 +1988,11 @@ TEST_P(
   // true, inserts into such unpartitioned tables are not allowed.
   //
   // We assert that an error is thrown in this case.
+  std::unordered_map<std::string, std::string> propFromFile{
```

Comment:
Shall we have a test case here and expect a success if immutable-partitions is false?